Dockerfile

```diff
@@ -1,7 +1,5 @@
 FROM zmkfirmware/zmk-build-arm:2.4
```

Comment:
I had to apply https://github.com/KinesisCorporation/Adv360-Pro-ZMK/pull/25 for v2 branch

Reply:
same, i just omitted it here since there's an open PR for it.

Reply:
Pr has since been merged btw